vochain/account.go

```diff
@@ -87,46 +87,33 @@ func (a *Account) DelDelegate(addr common.Address) {
 // and updates the state with the new values (including nonce).
 // If origin address acc is not enough, ErrNotEnoughBalance is returned.
 // If provided nonce does not match origin address nonce+1, ErrAccountNonceInvalid is returned.
-// If isQuery is set to true, this method will only check against the current state (no changes will be stored)
-func (v *State) TransferBalance(from, to common.Address, amount uint64, nonce uint32, isQuery bool) error {
-	var accFrom, accTo Account
-	if !isQuery {
-		v.Tx.Lock()
-		defer v.Tx.Unlock()
-	}
-	accFromRaw, err := v.Tx.DeepGet(from.Bytes(), AccountsCfg)
-	if err != nil {
+func (v *State) TransferBalance(from, to common.Address, amount uint64, nonce uint32) error {
+	var accFrom, accTo *Account
+	var err error
+	if accFrom, err = v.GetAccount(from, false); err != nil {
```

Comment:
You don't need these `var`s:

```suggestion
	accFrom, err := v.GetAccount(from, false)
	if err != nil {
```

Reply:
joining the error check with the `if` can help reduce lines in some situations, but in this case it just doesn't help